README.md

```diff
@@ -1 +1,65 @@
-# zebra-backend
\ No newline at end of file
+[![Build Status][travis-badge]][travis-badge-url]
+[![Coverage Status][coveralls-badge]][coverage-badge-url]
+
+# Zebra Backend
+
+## Local configuration
+All required configuration is stored in *application-dev-sample.yml* file. Copy it do *application-dev.yml* and add proper configuration information.
+
+To make this file be loaded during startup you need to activate app profile *dev*. Easiest way in local setup is to set environment variable *SPRING_PROFILES_ACTIVE*.
```

Comment:
Added link to detailed Spring Boot documentation. I think it's more appropriate than explaining details here.